### DISCUSSION TITLE
为什么添加后站点里面看不到rss的种子之类的信息呢

### DISCUSSION THREAD

**Discussion by wintsa123** [jxxghp/MoviePilot#1651] [问题咨询]
![902bf239c3681410696f69dc88c825e2](https://github.com/jxxghp/MoviePilot/assets/47288630/57da61ed-e3f7-43c7-9e9c-da3067f5cf9a)

**Comment by Simle** [comment]
> ![902bf239c3681410696f69dc88c825e2](https://github.com/jxxghp/MoviePilot/assets/47288630/57da61ed-e3f7-43c7-9e9c-da3067f5cf9a)
> 
> 

关闭仿真

**Comment by wintsa123** [comment]
> 
> 关闭仿真
没有作用
